tests/python/gaia-ui-tests/gaiatest/tests/manifest.ini

```diff
@@ -47,7 +47,7 @@ smoketest = false
 # Is a sanity test for core device features
 sanity = false
 
-# The test requires the use of external/internet resources
+# The test requires the use of internet resources or the Plivo server
```

Comment:
FYI for future potential reader, we send HTTP requests to Plivo to trigger the phone calls. That's why Plivo is considered as an external internet resource.